src/app/states/conversation.service.ts

```diff
@@ -51,18 +53,33 @@ export class ConversationService {
   highlightSubject = new BehaviorSubject<CompletedConversationMessage | null>(
     null,
   );
+  conversationId = Date.now().toString();
   ongoingConversations: OngoingConversationRecognition[] = [];
 
   messages$ = this.messagesSubject.asObservable();
 
   highlight$ = this.highlightSubject.asObservable();
+  selectedModel = '?';
 
-  constructor(private openAI: OpenAiService, private speaker: SpeakerService) {
+  constructor(
+    private openAI: OpenAiService,
+    private speaker: SpeakerService,
+    firebase: FirebaseService,
+  ) {
     this.openAI.state$.subscribe((state) => {
       this.clear(state);
     });
 
+    this.messages$.subscribe((messages) => {
+      if (messages.length > 1) {
```

Comment:
clear btn resets conversation and should reset the conversationId probably

Reply:
Good catch! Updated in 045df7.